src/main/flight/imu.c

```diff
@@ -230,12 +230,7 @@ STATIC_UNIT_TESTED void imuMahonyAHRSupdate(float dt, float gx, float gy, float
         // Compute heading vector in EF from scalar CoG. CoG is clockwise from North
         // Note that Earth frame X is pointing north and sin/cos argument is anticlockwise
         const fpVector2_t cog_ef = {.x = cos_approx(-courseOverGround), .y = sin_approx(-courseOverGround)};
-#define THRUST_COG 1
```

Comment:
Have a look at line 243
```c
#if THRUST_COG
        // increase gain for small tilt (just heuristic; sqrt is cheap on F4+)
        ez_ef /= sqrtf(heading_mag);
#endif
```
Is this supposed to run or not?

Reply:
oh damn good catch. @ledvinap ?